src/scripts/main.js

```diff
@@ -1,7 +1,92 @@
-'use strict';
+const Game = require('../modules/Game.class');
+const game = new Game();
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const container = document.querySelector('.container');
+const gameField = document.querySelector('.game-field');
+const button = container.querySelector('.button');
+const gameScore = container.querySelector('.game-score');
 
-// Write your code here
+const fieldCells = [...gameField.querySelectorAll('.field-row')].map((row) => [
+  ...row.children,
+]);
```

Comment:
The current implementation of selecting field cells is not very efficient because it queries the DOM for each row and then for each child. Consider storing a reference to the cells directly if possible, or using a more efficient selector.